core/src/main/scala/org/apache/spark/Accumulators.scala

```diff
@@ -271,6 +307,65 @@ class Accumulator[T] private[spark] (
   }
 }
 
+case class UpdateInfo private[spark](val rddId: Int, splitId: Int)
```

Comment:
I think it's better to use the same name in all usage of UpdateInfo class. Here you use splitId, and at method addInPlace you use partitionId.